version/version.go

```diff
@@ -64,6 +66,23 @@ func (v *Version) AtLeast(minVersion string) bool {
 	return v.Compare(minVersion) <= 0
 }
 
+func (v *Version) GetMajor() (string, error) {
+	return v.extractPosition(0)
+}
+func (v *Version) GetMinor() (string, error) {
+	return v.extractPosition(1)
+}
+func (v *Version) GetPatch() (string, error) {
+	return v.extractPosition(2)
+}
+
+func (v *Version) extractPosition(index int) (string, error) {
+	if semver.IsValid(v.version) {
```

Comment:
Shouldn't be "not"?
```suggestion
	if !semver.IsValid(v.version) {
```

Looks like IsValid expects the version to start with "v"